hil_starter.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+hil_dir=$1
+
+if [ "$#" -ne 1 ] || ! [ -d "$1" ]; then
+    echo "Usage: $0 path_to_HIL_directory" >&2
+    exit 1
+fi
+
+start_hil_server(){
```

Comment:
@vsemp this script seems to require the existence of HIL in some directory on the user's local machine. I think in previous discussions it was decided that we should keep this out of QUADS? Perhaps you could put this in HIL instead? Then you could still use this script to automate running the servers, but would not be adding any HIL dependencies to QUADS.